script/testing/oltpbench/configs/end_to_end_performance/tpcc.json

```diff
@@ -12,5 +17,8 @@
                 "client_time": 60
             }
         }
-    ]
+    ],
+    "env":{
+        "log_disk":"RAM disk"
```

Comment:
I called this `wal_device` in my code. I'm fine changing it to wal_disk if you think that is better but I think we should specify which log it is

Reply:
I like `wal_device`. I will change it in my code.